assets/javascripts/running.js

```diff
@@ -8,17 +8,38 @@ var liveViewElements = [];
 // (initialized in initLivelogAndTerminal())
 var logElements;
 
+// Reload broken thumbnails (that didn't exist yet when being requested) every 7th time
```

Comment:
Why every seventh time? Also, the previous code said every 7 seconds. I'm not sure either makes sense to me.

Reply:
I'm just outsourcing this code into a separate function. I actually prefer to focus on one aspect of the page at a time and not try to improve everything at once.

I've adjusted the comment because it was incorrect. This function ensures by no means that the broken thumbnails are reloaded every 7 seconds. They are just reloaded every 7th time the code reaches that point. This is very arbitrary.

There are also further problems in the code below. I haven't fixed them for the same reason and just aimed for not making things worse. I'll add these to @okurz "collection ticket".

Reply:
And the additional call to that function my change introduces does not make use of that weird "7th time" condition. So I don't think I'm making things worse here.

Reply:
Sounds fair.